src/controllers/controlpickermenu.cpp

```diff
@@ -153,6 +153,7 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
     QMenu* eqMenu = addSubmenu(tr("Equalizers"));
     const int kNumEqRacks = 1;
     const int iNumDecks = ControlObject::get(ConfigKey("[Master]", "num_decks"));
+    const int iNumPreviewDecks = ControlObject::get(ConfigKey("[Master]", "num_preview_decks"));
```

Comment:
will remove this. not necesary anymore since 'addDeckAndPreviewDeckControl()' is used for star COs below